### PR TITLE
[AXON-23] fix: update the bitbucket-pipelines schema

### PR DESCRIPTION
**Pull request by sdzh-atlassian**
### What is this?

This is the change to update the schema for `bitbucket-pipelines.yaml` to what seems to be its latest publicly available version, straight from Bitbucket folks :) 

Sources:
 * https://community.atlassian.com/t5/Bitbucket-questions/Scheme-URL-of-bitbucket-pipelines-yml/qaq-p/834807
 * https://api.bitbucket.org/schemas/pipelines-configuration

### How was this tested?

```
npm run lint # OK
npm run test # OK
npm run extension:install # clicked around, made sure the pipeline files are validated properly
```

An example with a random new property
| Before | After |
|--------|--------|
| image | image |

### CODE REVIEW

**Comment by sdzh-atlassian** [comment]
I'm kind of trying out the daisy-chain PR startegy since I've never used it in github before 😉 Let's see what happens when the other PR is merged

File: resources/schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Source of the pipeline schema:
+https://api.bitbucket.org/schemas/pipelines-configuration
```
Comment (bwieger-atlassian-com) [inline resources/schemas/README.md:2]: Great note!

**Review by bwieger-atlassian-com — APPROVED** [review]
LGTM

**Comment by sdzh-atlassian** [comment]
Hmmm, looks like the daisy chain thing doesn't work on GitHub - or maybe the merge type was wrong on the other PR? 🤔

**Comment by sdzh-atlassian** [comment]
And I guess this is why lol:
image

**Review by bwieger-atlassian-com — APPROVED** [review]

**Pull request merged**